### ISSUE TITLE
Can asp.net core 3.0 support [Autowired] Attribute to implement Dependency injection

### ISSUE THREAD

**Issue by yunduansing** [aspnet/AspNetCore#14450]
Can asp.net core 3.0 support [Autowired] Attribute to implements Dependency injection(just like Spring Boot @Autowired)？
eg:
`
[Autowired(LifeTime=Sington)]
private readonly IUserService userService{set;get;}`
Only construct inject is Not very friendly.

**Comment by davidfowl** [comment]
No, but you can use a dependency injection container that does support property injection https://github.com/aspnet/Extensions/tree/master/src/DependencyInjection#using-other-containers-with-microsoftextensionsdependencyinjection

**Comment by yunduansing** [comment]
> No, but you can use a dependency injection container that does support property injection https://github.com/aspnet/Extensions/tree/master/src/DependencyInjection#using-other-containers-with-microsoftextensionsdependencyinjection

Have you  plan to provide official  property injection in asp.net core 3.x in the future.
